.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+on: [push, pull_request]
+
+jobs:
+  android-bin:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: seanmiddleditch/gha-setup-ninja@v3
+      - run: bazel build -c opt :encoder_main --config=android_arm64
+      - run: bazel build -c opt :decoder_main --config=android_arm64
+
+  android-app:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: seanmiddleditch/gha-setup-ninja@v3
+      - name: Build Android app
+        run: bazel build android_example:lyra_android_example --config=android_arm64 --copt=-DBENCHMARK
+      - name: Upload Android APK
+        uses: actions/upload-artifact@v2
+        if: github.repository_owner == 'lyra'
+        with:
+          path: bazel-bin/android_example/lyra_android_example.apk
+      - name: Upload Android APK on release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: bazel-bin/android_example/lyra_android_example.apk
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I disabled both. If you want to enabled it in the future, remove the ` && false` part.

Reply:
I wasn't clear, sorry.  Please remove the entire upload sections.

Reply:
To confirm, you never want to use anything like this? Otherwise removing it will require it to be rewritten at some time.

With `if: false` that step will never run on the Google repo without being approved/commited by a Collaborator, but yet can be enabled easily when desired. I have [tested it](https://github.com/EwoutH/lyra/releases/tag/v0.0.1-test1) (on my fork) by the way.

Could you explain your reasoning for removing it alltogether, I don't understand it yet, sorry.

Reply:
Thanks for asking.  The main reasoning is to slim this PR down to the build verification functionality, and to avoid having inactive code.  There are also some security and legal issues we would have to look into if we add secrets.GITHUB_TOKEN, and until then we will prefer manual handling of release artifacts.

Reply:
Thanks for the explanation. I removed both the upload and deploy steps for now.